src/__tests__/loginRedux.test.js

```diff
@@ -1,6 +1,4 @@
 /* eslint-disable no-console */
```

Comment:
could you remove this line as you are not using console in the file

Reply:
I have removed it